app/dashboard/models.py

```diff
@@ -2993,7 +2996,7 @@ def to_dict(self, activities=True, leaderboards=True, network=None, tips=True):
             works_with_org = self.get_who_works_with(work_type='org', bounties=orgs_bounties)
 
         total_funded = funded_bounties.count()
-        total_fulfilled = fulfilled_bounties.count()
+        total_fulfilled = fulfilled_bounties.count() + self.tips.count()
```

Comment:
i wonder if we shouldnt update `total_funded` too while were at it...

Reply:
`total_funded` should already be correct. `total_funded` is communicated on the profile as the number of bounties the funder has opened and is funded. If we were to add how much they tipped per bounty, this number would be higher than the number of bounties they opened, as they can tip multiple people per bounty. However, since receiving a tip as a contributor does not count as a completed bounty by the `Bounty` object, `total_fulfilled` needed to be updated with tips count.